src/freenet/node/LocationManager.java

```diff
@@ -190,6 +190,9 @@ public void run() {
             @Override
             public void run() {
                 node.ticker.queueTimedJob(this, DAYS.toMillis(1));
+                if (swappingDisabled()) {
```

Comment:
Should not the job be queued _only_ if opennet is enabled?, if so we'll not would have to worry about this check (it will not hurt though). Then on the OpennetManager start up "notify" the LocationManager to start the job.

Reply:
In that case the job would be missing, if someone switches off opennet.